### PR TITLE
chore: update uno

### PR DESCRIPTION
**Pull request by Robert-Louis**
Update uno fixing the layout trigger arrange phase multiple time
https://github.com/unoplatform/uno/issues/14373

### CODE REVIEW

File: reference/TubePlayer/Directory.Packages.props [hunk 1]
```diff
@@ -29,7 +29,7 @@
     <PackageVersion Include="Uno.Wasm.Bootstrap" Version="8.0.4" />
     <PackageVersion Include="Uno.Wasm.Bootstrap.DevServer" Version="8.0.4" />
     <PackageVersion Include="Uno.Wasm.Bootstrap.Server" Version="7.0.31" />
-    <PackageVersion Include="Uno.WinUI" Version="5.0.64" />
+    <PackageVersion Include="Uno.WinUI" Version="5.1.0-dev.1268" />
```
Comment (agneszitte) [inline reference/TubePlayer/Directory.Packages.props:32]: @nickrandolph, @jeromelaban 
I think the other Uno.WinUI versions should match the same version as well (Uno.WinUI....). 
Also, the fix (https://github.com/unoplatform/uno/pull/14902) should be backported no ? 
So we can use a proper stable version for the TubePlayer sample app but also the same version update will be needed for the SimpleCalc sample app (cf. screenshot and details here: https://github.com/unoplatform/uno.toolkit.ui-private/issues/7)

**Review by agneszitte — CHANGES_REQUESTED** [review]

File: reference/TubePlayer/Directory.Packages.props (hunk 1 above)
Comment (Youssef1313) [reply reference/TubePlayer/Directory.Packages.props:32]: We shouldn't be using dev/prerelease versions. We should either wait for 5.1 stable or backport.

Comment (jeromelaban) [reply reference/TubePlayer/Directory.Packages.props:32]: Indeed, let's wait for the 5.1 release. Let's not backport either for now, it's a very significant change.

**Review by nickrandolph — CHANGES_REQUESTED** [review]
Please don't use dev versions in samples repository

**Comment by agneszitte** [comment]
> Please don't use dev versions in samples repository

@Robert-Louis  Now that 5.1 stable is available can you update your PR please so we can have the update for 5.1 stable for **TubePlayer** but also for **SimpleCalc** (cf. screenshots in https://github.com/unoplatform/Uno.Samples/pull/624)

Thanks in advance

**Review by eriklimakc — APPROVED** [review]
Tested all 4 versions of SimpleCalc and the TubePlayer app. No regression found, the apps are working as expected.

**Review by agneszitte — APPROVED** [review]

**Comment by agneszitte** [comment]
> Tested all 4 versions of SimpleCalc and the TubePlayer app. No regression found, the apps are working as expected.

Thank you so much for the quick sanity check @eriklimakc, really appreciated!

**Pull request merged**